webassembly/CMakeLists.txt

```diff
@@ -1,4 +1,12 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.17) # 3.17 required for CMP0099
+
+# Need https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10551
+if(NOT VTK_VERSION VERSION_GREATER_EQUAL 9.3.20231007)
```

Comment:
Do this need to actually be even newer ?

Reply:
Let's change that when Jaswant's patch is merged

Reply:
fair enough